.github/actions/deploy-firebase/action.yml

```diff
@@ -47,7 +47,13 @@ runs:
         identity-provider: ${{ inputs.identity_provider }}
         service-account-email: ${{ inputs.service_account_email }}
         project-id: ${{ inputs.project_id }}
-        prebuild-command: cd functions && python -m venv venv && source venv/bin/activate && pip install -r requirements.txt
+        prebuild-command: |
+          cd functions/python
+          python -m venv venv
+          source venv/bin/activate
+          pip install -r requirements.txt
```

Comment:
does cloud build not do this when deploying the function?

Reply:
I'm not sure. It was like this before this PR.

Reply:
oh i see it's just a formatting change